### ISSUE TITLE
Missing relation in meiotic chromosome segregation?

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3923]
I am wondering why 

FYPO:0005634 | equational sister chromatid separation during meiosis I
A cellular process phenotype in which sister kinetochores become separated during the first meiotic nuclear division, and sister chromatids subsequently separate to result in equational segregation.

isn't a child of

FYPO:0000678 | unequal homologous chromosome segregation
A cellular process phenotype in which homologous chromosomes are not segregated equally to both spindle poles in the first meiotic nuclear division.

**Comment by mah11** [comment]
I guess it wasn't made clear when FYPO:0005634 was requested & added that equational segregation at MI would always be unequal. If that is the case, I can add a link; if there are ever any exceptions, I can't.

**Comment by ValWood** [comment]
Well isn't it always "unequal"  if a sister chromatid  splits in two at meiosis I?

Or,  if it isn't there is a problem anyway with 

unequal homologous chromosome segregation
because  FYPO:0005634 | equational sister chromatid separation during meiosis I
is definitely a type of nondisjuction.

I think it is the only type I came across so far. I  don't remember  a situation where both homologs  get separated to the same side in meiosis I although I guess it can happen.

**Comment by mah11** [comment]
> Well isn't it always "unequal" if a sister chromatid splits in two at meiosis I?

If *a [single] chromatid* "splits" that's a different problem - breaking a chromosome.

If sister chromatids separate from each other during meiosis I, it's very unlikely that homologous chromosome segregation would end up equal (that would require both chromatids to go to one pole, and both sister chromatids of the homolog to go to the other pole, in spite of the premature kinetochore dissociation ... and for that to happen for all homologous pairs). But biology being the heap of exceptions that it is, I wanted to make sure.

> I don't remember a situation where both homologs get separated to the same side in meiosis I although I guess it can happen.

I don't know how frequently it happens in fission yeast, but it abso-bleedin'-lutely happens a lot in bigger eukaryotes. Trisomy 21 anyone? 

This situation also fits the "nondisjunction" description better than cases where sister kinetochores dissociate too early. It's also probably what I had in mind when I wrote the FYPO:0000678 text def (although it's been so long I won't say definitely), and certainly what I was thinking when I made the "nondisjunction" synonyms exact. Looking at it again now, I ought to make the text def more explicitly broad, and change the synonyms scopes to related.
